serviceinsight/sequence-diagram/index.md

```diff
@@ -1,114 +1,115 @@
 
-Note: The order of operations is critical to the correctness of the sequence diagram. When the endpoints are running on more than one machine, it is crucial that the machines have synchronised clocks. This can be done using an NTP server.
+Note: The order of operations is critical to the correctness of the sequence diagram. When endpoints are running on more than one machine, it is crucial that those machines have synchronised clocks. This can be done using an NTP server.
 
 ## What is on the diagram
 
 ### Endpoints and lifelines
 
 ![Endpoint](endpoint.png)
 
-Each endpoint involved in the conversation is represented by a gray box along the top of the diagram. Hover on the endpoint to get additional information such as the host that the endpoint is running on. Extending from the bottom of each endpoint is a (life)line that shows time flowing from top to bottom.
+Each endpoint involved in the conversation is represented by a gray box along the top of the diagram. Hover on the endpoint to get more information such as the host that the endpoint is running on. Extending from the bottom of each endpoint is a (life)line that shows time flowing from top to bottom.
 
 
 ### Start of conversation
 
 ![Start of conversation marker](start-of-conversation.png)
 
-Each conversation is initiated by a single command or event. This message is often triggered by some action which is external to the system such as a user clicking a Submit Order button on a website. The metadata used to generate the diagram does not include the trigger in this case but it does include the endpoint that sent or published the message. This is represented by a Start of Conversation marker on the endpoint lifeline.
+Each conversation starts with a single command or event. This message is often triggered by some action external to the system such as a user clicking a Submit Order button on a website. The metadata used to generate the diagram does not include the trigger such cases but it does include the endpoint that sent or published the message, represented by a Start of Conversation marker on the endpoint lifeline.
 
-NOTE: Not every sequence diagram will have the start of conversation marker. This can happen if the conversation started a long time ago and the initiating message has expired. It can also happen if the number of messages in the conversation is very large. In this case, ServiceInsight will only get 50 messages from the conversation and this may not include the initiating message.
+NOTE: Not every sequence diagram will have a Start of Conversation marker. This can happen if the conversation started a long time ago and the initiating message has expired. It can also happen if the number of messages in the conversation is very large. The diagram will only show get the last 50 messages from the conversation and this may not include the initiating message.
```

Comment:
The first part of the paragraph seems a bit misleading. Could we rephrase? Something like:

>Start of Conversation marker might not be always visible in a sequence diagram. This can happen...